### DISCUSSION TITLE
The problem of IMU noise model units

### DISCUSSION THREAD

**Discussion by zhaoxin0716** [borglab/gtsam#1733] [Q&A]
Hello everyone, sorry to bother you. 
I am not very clear about the noise model units of IMU in GTSAM. 
For example, in the example of IMUKittiExampleGPS.cpp, I am not sure the unit of accelerometer_sigma，Gyroscope_sigma，Integration_sigma，Accelerometer_bias_sigma，Gyroscope_bias_sigma.
Perhaps Gyroscope_bias_sigma is rad/s,Accelerometer_bias_sigma is m/s^2.
If anyone knows about this, I would greatly appreciate it。
![QQ截图20240312221340](https://github.com/borglab/gtsam/assets/100476821/a9bf291a-1555-425c-b0a9-049f86a2e6da)
![图片1](https://github.com/borglab/gtsam/assets/100476821/8ef34991-31f2-40df-8ac6-08e1c688a173)

**Comment by scott-robotics** [comment]
The units of sigma (standard deviation) are the same as the units of the original data. So your assumption is correct.
